internal/commands/push/command.go

```diff
@@ -153,12 +186,12 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 
 	var uploadPathDependencies string
 	var dependenciesDiffs realm.DependenciesDiff
-	if cmd.inputs.IncludeDependencies {
-		appDependencies, err := local.FindAppDependencies(app.RootDir)
+	if cmd.inputs.IncludeNodeModules || cmd.inputs.IncludePackageJSON {
```

Comment:
```suggestion
	if cmd.inputs.IncludeDependencies || cmd.inputs.IncludeNodeModules || cmd.inputs.IncludePackageJSON {
```

Reply:
ill put the `cmd.inputs.IncludeDependencies` at the end, so it align with other files.